dotCMS/src/integration-test/java/com/dotmarketing/portlets/workflows/actionlet/SaveContentActionletTest.java

```diff
@@ -72,11 +75,22 @@ private TestCase(
 
     @DataProvider
     public static Object[] usersAndContentTypeWithoutHostField() throws Exception {
+        IntegrationTestInitService.getInstance().init();
+
+        final Optional<WorkflowAction> optionalSaveAction = getWorkflowActionByName("Save");
+        final Optional<WorkflowAction> optionalPublishAction = getWorkflowActionByName("Publish");;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [An empty statement (semicolon) not part of a loop](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5397063)